### PR TITLE
bump up the version

### PR DESCRIPTION
**Pull request by asad-rafter**

### CODE REVIEW

File: Cloudinary-pwa-kit/documentation/Cloudinary-pwa-integration-guide.docx
Comment (yuval-cloudinary) [inline Cloudinary-pwa-kit/documentation/Cloudinary-pwa-integration-guide.docx:1]: What was modified in the docs?

Comment (asad-rafter) [reply Cloudinary-pwa-kit/documentation/Cloudinary-pwa-integration-guide.docx:1]: we have changed in footer, so added that change in docs

Comment (yuval-cloudinary) [reply Cloudinary-pwa-kit/documentation/Cloudinary-pwa-integration-guide.docx:1]: what was the change exactly?

Comment (asad-rafter) [reply Cloudinary-pwa-kit/documentation/Cloudinary-pwa-integration-guide.docx:1]: On page 12, section 3 Salesforce Commerce Cloud PWA KIT, File 1 (packages\pwacloudinary\overrides\app\components\footer\index.jsx) has changed.

**Pull request merged**